src/components/Identity/index.js

```diff
@@ -58,7 +58,7 @@ const createIdentity = ({ config, logger, cookieJar }) => {
         return optIn.whenOptedIn().then(() => {
           if (
             !alreadyQueriedForIdSyncs &&
```

Comment:
Do you want to change some of these variable names, or does it make sense to leave them as "syncs"?

Reply:
I think it makes sense to leave them.